packages/server/src/routes/me/index.ts

```diff
@@ -70,27 +70,48 @@ export const sendInvite = (
     // If there is already a videoToken, give them the deeplink format.
     const inviteLink = videoToken
       ? new URL(
-          `/invite/referrer=${loggedInUsername}&token=${videoToken}`,
+          `/invite?r=${loggedInUsername}&t=${videoToken}`,
           `https://raha.app`
         ).toString()
       : new URL(`/m/${loggedInUsername}/invite`, config.appBase).toString();
 
+    const webInstructionsText = `Visit ${inviteLink} to join Raha!`;
+    const webInstructionsHtml = `<span><strong>Visit <a href="${inviteLink}">${inviteLink}</a> to join Raha!</strong></span>`;
+
+    const mobileInstructionsText =
+      "1. Download the app:\n" +
+      "  Android: https://play.google.com/store/apps/details?id=app.raha.mobile" +
```

Comment:
Should we make the text  between the two versions match?

Reply:
I think the text version is easier to read with iOS/Android separated

Reply:
Matched the iOS language though!